src/helm/benchmark/run.py

```diff
@@ -90,16 +92,16 @@ def run_benchmarking(
     with htrack_block("run_specs"):
         for run_spec in run_specs:
             hlog(run_spec)
-
-    runner = Runner(
-        execution_spec,
-        output_path,
-        suite,
-        skip_instances,
-        cache_instances,
-        cache_instances_only,
-        skip_completed_runs,
-        exit_on_error,
+    runner_cls = SlurmRunner if use_slurm_runner else Runner
```

Comment:
If we make a `SlurmRunner`, maybe `Runner` should be renamed to `SimpleRunner`?  Not sure if we plan to have more runners or is it just going to be these two?